pom.xml

```diff
@@ -59,8 +59,10 @@
         <commons-io.version>2.4</commons-io.version>
         <mockito.version>1.10.19</mockito.version>
         <slf4j.version>1.7.5</slf4j.version>
-        <licenses.version>3.3.0-SNAPSHOT</licenses.version>
+	<licenses.name>Apache License 2.0</licenses.name>
```

Comment:
Hmm, still seems off based on what I'm seeing in this diff? `project.package.home` and the entire profile below look wonky too.